### PR TITLE
Add all arm request related files to Matser

### PR DESCRIPTION
**Pull request by Chris-MS-896**
Add 18 files related to arm request files to Matser

### CODE REVIEW

File: src/Utils/arm/request.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export async function armRequest<T>({
   const response = await window.fetch(url.href, {
     method,
     headers: {
-      Authorization: userContext.authorizationToken
+      Authorization: !userContext.authorizationToken ? "" : userContext.authorizationToken
```
Comment (southpolesteve) [inline src/Utils/arm/request.ts:69]: This is not correct. We must always have an authorization token set. Can we throw an error earlier in the function?

Something like: `if (!userContext.authorizationToken) { throw new Error("No token provided") }`

File: src/Utils/arm/request.ts [hunk 2]
```diff
@@ -100,7 +100,7 @@ export async function armRequest<T>({
 async function getOperationStatus(operationStatusUrl: string) {
   const response = await window.fetch(operationStatusUrl, {
     headers: {
-      Authorization: userContext.authorizationToken
+      Authorization: !userContext.authorizationToken ? "" : userContext.authorizationToken
```
Comment (southpolesteve) [inline src/Utils/arm/request.ts:103]: Same as other one

**Review by southpolesteve — CHANGES_REQUESTED** [review]

File: src/Utils/arm/request.ts (hunk 1 above)
Comment (Chris-MS-896) [reply src/Utils/arm/request.ts:69]: OK,I will fix that

File: src/Utils/arm/request.ts (hunk 2 above)
Comment (Chris-MS-896) [reply src/Utils/arm/request.ts]: I make the unit test and change in minor changes

**Review by ato9000 — APPROVED** [review]

**Review by southpolesteve — APPROVED** [review]

**Pull request merged**